src/Matrix.cc

```diff
@@ -2492,6 +2493,18 @@ NAN_METHOD(Matrix::MeanWithMask) {
   info.GetReturnValue().Set(arr);
 }
 
+NAN_METHOD(Matrix::Mean) {
+  SETUP_FUNCTION(Matrix)
+
+  cv::Scalar means = cv::mean(self->mat);
+  v8::Local<v8::Array> arr = Nan::New<Array>(3);
```

Comment:
Actually means return a 4 element scalar and not 3. The MeanWithMask example I followed is incorrect too. Let me fix it to return 4 elements.